ddtrace/span.py

```diff
@@ -477,7 +479,7 @@ def _remove_exc_info(self):
     def pprint(self):
```

Comment:
probably a bit of a misnomer now 😆

Reply:
Oh actually this is a public API change so we should have a release note for it (not that anyone should realistically be depending on the output...